styles/main.css

```diff
@@ -73,15 +85,18 @@ ul {
 body{padding-top: 0;
      padding-bottom: 0;
      }
-.mugshot{box-shadow: 2px 2px 2px 3px rgba(0, 0, 0, 0.5);
-        }
-hr {
-    width:50%;
+.mugshot img {
+    box-shadow: 2px 2px 3px rgba(0, 0, 0, 0.5);
+}
+footer hr {

```

Comment:
Corrected selector (using `footer hr` selects ONLY the `hr` elements inside `footer` elements as per the lab